src/caselawclient/identifier_resolution.py

```diff
@@ -0,0 +1,31 @@
+import json
+from typing import NamedTuple
+
+from caselawclient.models.documents import DocumentURIString
+from caselawclient.xquery_type_dicts import MarkLogicDocumentURIString
+
+
+class IdentifierResolution(NamedTuple):
```

Comment:
Can we pop some docstrings against this, just to explain what it's doing?

Reply:
Done